packages/@lwc/errors/src/compiler/error-info/template-transform.ts

```diff
@@ -503,7 +503,7 @@ export const ParserDiagnostics = {
     LWC_DOM_INVALID_IN_LIGHT_DOM: {
         code: 1135,
         message:
-            "Invalid directive 'lwc:dom' on element. 'lwc:dom' is not supported in Light DOM components.",
+            "Invalid directive 'lwc:dom' on element {0}. 'lwc:dom' is not necessary in Light DOM components.",
```

Comment:
"Supported" makes it sound like you can't get the same functionality with light DOM components. Whereas the truth is you can just delete it for light DOM components.